chain/vm/fvm.go

```diff
@@ -65,11 +65,15 @@ func (t *FvmExecutionTrace) ToExecutionTrace() types.ExecutionTrace {
 		Error:      t.Error,
 		Duration:   0,
 		GasCharges: nil,
-		Subcalls:   make([]types.ExecutionTrace, len(t.Subcalls)),
+		Subcalls:   nil, // Should be nil when there are no subcalls for backwards compatibility
```

Comment:
You should be able to drop this line (and the two above) -- the default values will be 0 and nil.